sdk/formrecognizer/azure-ai-formrecognizer/samples/v3.2-beta/async_samples/sample_analyze_document_async.py

```diff
@@ -0,0 +1,173 @@
+# coding: utf-8
+
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# --------------------------------------------------------------------------
+
+"""
+FILE: sample_analyze_document_async.py
+
+DESCRIPTION:
+    This sample demonstrates how to extract general document information from a document
+    given through a file.
+
+    Note that selection marks returned from begin_analyze_document() do not return the text associated with
+    the checkbox. For the API to return this information, build a custom model to analyze the checkbox and its text.
+    See sample_build_model_async.py for more information.
```

Comment:
let's nix this comment... more of a v2 limitation. You might be able to infer the text associated with the checkbox now that everything is included in reading order in the content string.